src/xbot/common/command/BaseRobot.java

```diff
@@ -34,6 +36,8 @@
     protected Injector injector;
     
     protected Command autonomousCommand;
+    
+    protected Vector<BaseSubsystem> telemetryUpdatingSubsystems = new Vector<BaseSubsystem>();
```

Comment:
Is there a reason that this is a `Vector`? They were deprecated in Java 2 IIRC. It seems like this should be a normal `ArrayList`.